.github/workflows/docs-check.yml

```diff
@@ -92,8 +92,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Build draft PDF
-        uses: openjournals/openjournals-draft-action@master
+      - name: Build PDF
+        uses: openjournals/openjournals-draft-action@v.1.0
```

Comment:
Is this correct?
```suggestion
        uses: openjournals/openjournals-draft-action@v1.0
```

Reply:
Never mind 😅 https://github.com/openjournals/openjournals-draft-action/tree/v.1.0

Reply:
thought the same but their tag is just strange :D